spec/fixtures/constants.rb

```diff
@@ -106,6 +106,13 @@
     cd /Users/musalj/code/OSS/Alcatraz
     /usr/bin/touch -c /Users/musalj/Library/Developer/Xcode/DerivedData/Alcatraz-aobuxcinaqyzjugrnxjjhfzgwaou/Build/Products/Debug/AlcatrazTests.octest
 )
+
+SAMPLE_WRITE_FILE = %Q(
+write-file /Users/me/myproject/Build/Intermediates/Pods.build/Debug-iphonesimulator/Pods-AFNetworking.build/Objects-normal/x86_64/Pods-AFNetworking.LinkFileList
+)
+
+SAMPLE_WRITE_AUXILIARY_FILES = %Q(Write auxiliary files)
```

Comment:
Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.